R/tidy_observation.R

```diff
@@ -1,13 +1,76 @@
 #' tidy_observation
 #'
-#' Select the Desired Observation
+#' This function is used to prepare the observation datasets for the use in function tidy_ae_table().
```

Comment:
just add code formatting 
```suggestion
#' This function is used to prepare the observation datasets for the use in function `tidy_ae_table()`.
```